app/views/providers/allocations/pick_a_provider.html.erb

```diff
@@ -0,0 +1,67 @@
+<%= content_for :page_title, "Pick a provider"%>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(initial_request_provider_recruitment_cycle_allocations_path(@provider.provider_code, @recruitment_cycle.year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+      <h1 class="govuk-heading-xl" data-qa="heading">Pick a provider</h1>
+    </legend>
```

Comment:
We should be using a `legend` here. Just a normal `h1` should do it

Reply:
this is from Find so it'll need to be fixed there too

Reply:
no wait thats rubbish, totally not true, ignore me

Reply:
the legend is now gone

Reply:
https://github.com/DFE-Digital/find-teacher-training/issues/288 created an issue for it